malep2007/landlord-backend
# Feature To Create Propert

## Description
This feature helps to create a property with some details like the name of property, location and description. 

## How do you test it?
- Clone the Repo, navigate to it and run `pip install -r requirements.txt`
- Run the server with `python manage.py runserver`
- Using insomnia or any GraphQL client, and navigate to the URL `http://localhost:8000/landlord/`
- paste this query on the body 
```
query{
   properties {
        id
        name
        description
    }
}
```
-  You should expect a response in the format of 
```
{
   data {
        properties : null
    }
}
```
